src/java/org/apache/cassandra/cql3/conditions/ColumnCondition.java

```diff
@@ -573,6 +576,38 @@ private static boolean mapAppliesTo(MapType<?, ?> type, Iterator<Cell<?>> iter,
         }
     }
 
+    private static boolean containsAppliesTo(CollectionType<?> type, Iterator<Cell<?>> iter, ByteBuffer value, Operator operator){
```

Comment:
Nit: the brace should be on the next line.

Reply:
@Ankou76ers friendly reminder